src/kvstore/CMakeLists.txt

```diff
@@ -3,10 +3,12 @@ add_library(
     Part.cpp
     RocksEngine.cpp
     PartManager.cpp
+    KVStore.cpp
     NebulaStore.cpp
     RocksEngineConfig.cpp
 )
-add_dependencies(kvstore_obj base_obj meta_client)
+add_dependencies(kvstore_obj base_obj meta_client hbasestore_obj)
```

Comment:
kvstore_obj should NOT depend on hbasestore_obj. 
They are on the same level.

Reply:
The hbasestore will be as a plugin.